### PR TITLE
Ovl_Dm_Nb OK

### PR DESCRIPTION
**Pull request by hensldm**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

Going to wait for more code decomp to document this.

### CODE REVIEW

File: src/overlays/actors/ovl_Dm_Nb/z_dm_nb.c [hunk 1]
```diff
@@ -9,7 +15,10 @@ void DmNb_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void DmNb_Update(Actor* thisx, GlobalContext* globalCtx);
 void DmNb_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+UNK_TYPE func_80C1DED0(DmNb* this, s32 arg1);
+void func_80C1DF18(DmNb* this, GlobalContext* globalCtx);
+void DmNb_UnkActorDraw(GlobalContext* globalCtx, s32 limbIndex, Actor* actor);
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_Dm_Nb/z_dm_nb.c:20]: Looks like those forward declarations are not needed.

**Review by AngheloAlf — COMMENTED** [review]
Looking good, just a minor comment

File: src/overlays/actors/ovl_Dm_Nb/z_dm_nb.c (hunk 1 above)
Comment (Kenix3) [reply src/overlays/actors/ovl_Dm_Nb/z_dm_nb.c:20]: I'm personally OK with verbose forward declarations. They read as a list of functions available to the Actor system.

**Review by Kenix3 — APPROVED** [review]

File: src/overlays/actors/ovl_Dm_Nb/z_dm_nb.c [hunk 2]
```diff
@@ -21,18 +30,78 @@ const ActorInit Dm_Nb_InitVars = {
     (ActorFunc)DmNb_Update,
     (ActorFunc)DmNb_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Dm_Nb_0x80C1DED0/func_80C1DED0.asm")
+extern FlexSkeletonHeader D_06008C40;
+
+// Probably the same struct as EnFgAnimation, need more info from func_8013BC6C
+static UNK_TYPE D_80C1E200[] = { 0x06000990, 0x3F800000, 0x0000FFFF, 0x00000000 };
```
Comment (Kenix3) [inline src/overlays/actors/ovl_Dm_Nb/z_dm_nb.c:37]: The object segment 06 pointer should still be a symbol at least.

**Review by Kenix3 — CHANGES_REQUESTED** [review]

File: include/functions.h [hunk 3]
```diff
@@ -1595,7 +1595,7 @@ void nop_800EDDB0(GlobalContext* globalCtx);
 // void func_800EDDBC(void);
 void func_800EDDCC(GlobalContext* globalCtx, u32 uParm2);
 void func_800EDE34(Actor* actor, GlobalContext* globalCtx, s32 param_3);
-// void func_800EDF24(void);
+void func_800EDF24(Actor* actor, GlobalContext* ctxt, u32 arg2);
```
Comment (engineer124) [inline include/functions.h:1598]:
```suggestion
void func_800EDF24(Actor* actor, GlobalContext* globalCtx, u32 arg2);
```
Just one minor suggestion from me. The rest looks good!

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**